ユーザーのログイン機能を実装するため

yukri1030 opened the pull request:
# WHAT
アプリ製作に必要なログインのviewを実装する

# WHY
アプリの機能を充実させるため

yukri1030 committed:
create pull request

yukri1030 committed:
サインイン画面のビューの実装

# WHAT
新規登録画面のビューを実装しました。
仮置きのルートとコントローラーを設定。
画面全体を一つのクラスで指定し、そこから、vhを用いて、ブロックを切り分けていきました。

# WHY
新規登録時のフロント画面が必要のため

bomber0522 commented on app/views/users/new.html.haml:
```diff
@@ -0,0 +1,32 @@
+      .contents__btn_google
+        = image_tag 'google_icon.png',size: "35x37" , class: "google-icon "
+        = link_to "Googleで登録"
+  .footer
```
footer は部分テンプレートで切り分けた方が良いかと思います＾＾

bomber0522 replied:
画面の画像も貼り付けた方が良いかと

yukri1030 replied:
修正しました！
画面の画像のURLも、貼付しました！

bomber0522 replied:
googleロゴは良いと思いますが、mailロゴとfacebookロゴが若干位置調整必要かも?

yukri1030 replied:
ありがとうございます！mailロゴ修正しました。facebookロゴはうまく行かず・・・

bomber0522 commented on app/views/users/new.html.haml:
```diff
@@ -0,0 +1,32 @@
+  .footer
+    .hosoku
+      %nav
+        %ul.clearfix
```
clearfixなど、様々なタグで使用されるスタイルはクラスを複数付与する形で対応するのではなくmixinで該当CSSで呼び出す形にするとなっているみたいなのでクラスのclearfixはmixinでできませんか？

yukri1030 replied:
claerfixをmixinで指定しました！

yukri1030 committed:
サインイン画面のビューの実装

# WHAT
footerを部分テンプレートとして切り分けた
@exrtend を使用した

# WHY
viewの修正を容易に行えるようにするため

yukri1030 reviewed (commented):
https://gyazo.com/b725e53be8b854323f94ac027df57ea1

yukri1030 committed:
サインイン画面の実装

# WHAT
 mixin　includeの使用

# WHY
ビューの修正を行いやすくするため

https://gyazo.com/3388385ddbb5523d76d90d629012ad9f

yukri1030 committed:
ビューの修正

# WHAT
フッタークラスの高さが大きすぎるため、修正しました。それに伴って全体的にビューを修正しました

メールのアイコンの位置を修正しました
フェイスブックアイコンも修正したかったのですが、うまくいきませんでした

# WHY
ビューのレイアウトを綺麗にするため

https://gyazo.com/06215fc6e357f3b5fd3b01c9ae948b8d

yukri1030 committed:
Merge remote-tracking branch 'origin' into user_log_in

yukri1030 committed:
デフォルトのビューの修正＆ レスポンシブ対応のscssの作成

# WHAT
新規作成画面のビューの高さを修正しました。（vh → px　にしました）
レスポンシブ対応のscss(_new.mq.scss)の作成をしました。
※検証ツールを用いて確認しましたが、自分のスマホにアクセスして確認はしていません。そのため、レイアウトの崩れがあれば随時直していきます）

# WHY
サインインにおいて、新規登録画面は必要なビュー。
レスポンシブ対応で、スマホにも対応させる。

yukri1030 commented:
レスポンシブ対応の画面です
[![Image from Gyazo](https://i.gyazo.com/4981bcbca295ff0286736fbf06967a07.png)](https://gyazo.com/4981bcbca295ff0286736fbf06967a07)

yukri1030 commented:
すみません。上記のメディアクエリ 対応のファイルは、修正中です。デフォルトのファイルのみ、レビューをお願いします

TOMARUMARU commented:
まずはconflictを解消しましょう

yukri1030 committed:
Merge branch 'master' into user_log_in

yukri1030 commented:
すみません。修正しました

yukri1030 commented:
すみません。修正しました
[![Image from Gyazo](https://i.gyazo.com/b725e53be8b854323f94ac027df57ea1.png)](https://gyazo.com/b725e53be8b854323f94ac027df57ea1)

shingoteshima commented on app/views/users/_footer.html.haml:
```diff
@@ -0,0 +1,15 @@
+      %small © 2019 Mercari
+    %br
+    %br
+    %br
```
brではなくmarginを使いましょう。

yukri1030 replied:
すみません。修正しました。

shingoteshima commented on config/routes.rb:
```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  get 'users/new'
```
resourcesで書きましょう。

yukri1030 replied:
すみません。修正しました。

yukri1030 committed:
ビューのレイアウトの調整

# WHAT
フッタークラスに、margin-bottomを追加
ルートをresources　で修正

# WHY
綺麗なレイアウトにしていくため

shingoteshima commented on config/routes.rb:
```diff
@@ -1,4 +1,4 @@
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root "products#index"
-end
+  resources :users  
```
onlyでアクション指定しましょう。

yukri1030 replied:
修正しました！

yukri1030 replied:
修正しました

yukri1030 committed:
アクション名の修正

＃WHAT
ルートのアクションをonlyで修正

# WHY
正規のアクションの指定

TOMARUMARU commented on app/assets/stylesheets/_new.mq.scss:
```diff
@@ -0,0 +1,111 @@
+      &__btn_google{
+        height: 110px;
+        width: 800px;
+        // display: flex;
```
不要なコメントアウトは消しましょう

yukri1030 replied:
削除しました

TOMARUMARU commented on config/initializers/carrierwave.rb:
```diff
@@ -1,20 +1,20 @@
+#     config.fog_directory  = 'freemarket-sample-50b'
+#     config.asset_host = 'https://s3-ap-northeast-1.amazonaws.com/freemarket-sample-50b'
+#   end
+# end
```
こちらのファイル全てコメントアウトされてますが理由はありますか？

yukri1030 replied:
コードを修正しました

yukri1030 committed:
carrierwave.rbの修正

# WHAT
carrierwave.rbを修正しました
また、ルートのコメントアウトを削除しました

# WHY
コードの修正により、可読性をあげる

TOMARUMARU reviewed (approved):
LGTM
